assimilation_code/modules/io/io_filenames_mod.f90

```diff
@@ -893,7 +926,20 @@ subroutine file_info_dump(file_info,context)
          write(*,'(A, I4,A,I4)') 'file_info%stage_metadata%io_flag(         ',i,'     ) ', &
                                 file_info%stage_metadata%io_flag(             i)
          write(*,'(A, I4,2A)')   'file_info%stage_metadata%copy_name(       ',i,'     ) ', &
-                                  file_info%stage_metadata%copy_name(i)
+                                  trim(file_info%stage_metadata%copy_name(    i))
+         write(*,'(A, I4,2A)')   'file_info%stage_metadata%long_name(       ',i,'   ) ', &
+                                  trim(file_info%stage_metadata%long_name(    i))
+
+         write(*,'(A, I4,A,L)')   'file_info%stage_metadata%clamp_vars(       ',i,'  ) ', &
```

Comment:
i haven't seen L in a format.  is it supported on all compilers we can test with?